allure/pytest_plugin.py

```diff
@@ -114,8 +114,10 @@ def pytest_runtest_protocol(self, item, nextitem):
         except AttributeError:
             # for doctests that has no `function` attribute
             description = item.reportinfo()[2]
+        # Mostly for markdown purposes
+        description = clean_docstring(description)
         self.test = TestCase(name='.'.join(mangle_testnames([x.name for x in parent_down_from_module(item)])),
-                             description=description,
+                             description=Description(value=description, type="markdown"),
```

Comment:
Lets have test for that stuff -- that it is actually recorded properly in the resulting XML